app/api/server/v1/users.js

```diff
@@ -402,7 +403,7 @@ API.v1.addRoute('users.setStatus', { authRequired: true }, {
 	},
 });
 
-API.v1.addRoute('users.update', { authRequired: true }, {
+API.v1.addRoute('users.update', { authRequired: true, twoFactorRequired: true }, {
```

Comment:
does this means that it will require two factor to update any user info? if so, should it be applied to [insertOrUpdateUser method](https://github.com/RocketChat/Rocket.Chat/blob/deca01221e20f48f6e89dd6a5324e650b609796e/app/lib/server/methods/insertOrUpdateUser.js#L7) as well as both basically call `saveUser()`?